filebeat/filebeat.template.json

```diff
@@ -1,9 +1,6 @@
 {
   "mappings": {
     "_default_": {
-      "_all": {
```

Comment:
We should probably now call this template `fielbeat.template.6x.json`. Otherwise I think it is confusing that the "default" template is not for the stable version.